tools/test/test_depsolve.py

```diff
@@ -1409,29 +1522,135 @@ def test_depsolve(tmp_path, repo_servers, dnf_config, detect_fn, with_sbom, test
                 assert res["solver"] == "dnf"
 
 
+@pytest.mark.parametrize("with_sbom", [False, True])
+@pytest.mark.parametrize("dnf_config, detect_fn", [
+    ({}, assert_dnf),
+    ({"use_dnf5": False}, assert_dnf),
+    ({"use_dnf5": True}, assert_dnf5),
+], ids=["no-config", "dnf4", "dnf5"])
+def test_depsolve_sbom(repo_servers, dnf_config, detect_fn, with_sbom):
+    try:
+        detect_fn()
+    except RuntimeError as e:
+        pytest.skip(str(e))
+
+    test_case = depsolve_test_case_basic_2pkgs_2repos
+    transactions = test_case["transactions"]
+    tc_repo_servers = get_test_case_repo_servers(test_case, repo_servers)
+    repo_configs = [gen_repo_config(server) for server in tc_repo_servers]
+
+    with TemporaryDirectory() as cache_dir:
```

Comment:
(super nitpick) we could just use the `tmp_path` fixture in the function arg and then we don't need with cache_dir  and indent  (here and in the other tests) which would be slightly shorter. But of course fine as is